can/can.yml

```
@@ -242,73 +236,73 @@ messages:
             - INVALID_STATE
             - LIMIT_EXCEEDED
 
-- dbwActive:
+- DBW_Active:
     id: 0x420
     cycletime: 100
 
     signals:
-      - Active:
+      - active:
           description: DBW is active.
           width: 1
 
-- dbwEnable:
+- DBW_Enable:
     id: 0x421
     cycletime: 100
 
     signals:
-      - Enable:
+      - enable:
           description: Enable DBW active.
           width: 1
 
-- dbwUpdater_Update_Trigger:
+- DBW_UpdaterUpdateTrigger:
     id: 0x4A0
     cycletime: 100
 
     template: dbwnodes
 
     signals:
-      - Trigger:
+      - trigger:
           description: Trigger the update.
           width: 1
-      - Filler1:
+      - filler1:
           description: Filler.
           width: 7
-      - Begin:
+      - begin:
           description: Begin the update.
           width: 1
 
-- dbwUpdater_Update_Data:
+- DBW_UpdaterUpdateData:
     id: 0x4B0
     cycletime: 100
 
     template: dbwnodes
 
     signals:
-      - Position:
+      - position:
           description: Index of 32-bit data chunk.
           width: 24
-      - Data:
+      - data:
           description: Image data.
           width: 40
 
-- dbwNode_Update_Response:
+- DBW_NodeUpdateResponse:
```

```suggestion
- NodeUpdateResponse:
```

It's ok, leave it for now. Will be reworked in bootloader PR